src/iOS/Avalonia.iOS/AvaloniaView.cs

```diff
@@ -99,6 +102,7 @@ public TopLevelImpl(AvaloniaView view)
                 {
                     view._topLevel.Padding = b ? default : _insetsManager.SafeAreaPadding;
                 };
+                _clipboard = new ClipboardImpl();
```

Comment:
and probably here

Reply:
iOS implementation doesn't have any class local objects, so it's essentially a static class.